apps/web/src/routes/(marketing)/(auth)/register/+page.svelte

```diff
@@ -24,6 +26,53 @@
 	} = superForm(data.registerForm, {
 		validators: formSchema
 	});
+
+	async function checkError() {
+		console.log(
+			$errors.display_name,
+			'form display name',
+			$errors.password,
+			'form password',
+			$errors.email,
+			'errors'
+		);
+		if (
+			$errors.display_name !== undefined ||
+			$errors.password !== undefined ||
+			$errors.email !== undefined
+		) {
+			toast.error('Please fill the form correctly');
+		} else {
+			if (form?.error) {
+				return toast.error(form?.error);
+			} else {
+				toast.success('Your account has been created', {
+					description: 'Check your email to activate your account'
+				});
+			}
+		}
+	}
+
+	let localErrorMessage = '';
+	$: localErrorMessage = form?.error;
+
+	async function chekcAutoReloadError() {
+		console.log('top:', localErrorMessage);
+		setTimeout(() => {
+			checkError();
+			// if (form?.error) {
+			// 	console.log('Form error:', form.error);
+			// 	toast.error(form.error);
+			// } else {
+			// 	console.log('No form error, showing success toast');
+			// 	toast.success('Your account has been created', {
+			// 		description: 'Check your email to activate your account'
+			// 	});
+			// }
```

Comment:
Commented code should be deleted before PR, it will remain in git history so you can get it back later